doc/developer-notes.md

```diff
@@ -362,29 +362,31 @@ If you have ccache enabled, absolute paths are stripped from debug information
 with the `-fdebug-prefix-map` and `-fmacro-prefix-map` options (if supported by the
 compiler). This might break source file detection in case you move binaries
 after compilation, debug from the directory other than the project root or use
-an IDE that only supports absolute paths for debugging.
+an IDE that only supports absolute paths for debugging (e.g. it won't stop at breakpoints).
 
-There are a few possible fixes:
+There are a few possible fixes (assuming the project root is `~/bitcoin`):
 
 1. Configure source file mapping.
 
-For `gdb` create or append to `.gdbinit` file:
+For `gdb` create or append to [`.gdbinit` file](https://sourceware.org/gdb/current/onlinedocs/gdb#gdbinit-man):
 ```
-set substitute-path ./src /path/to/project/root/src
+set substitute-path ./src ~/bitcoin/src
```

Comment:
My take was that `~/bitcoin/src` is easier to generalize to e.g. `C:\\Users\\Username\\Documents\\bitcoin\\src` than `/path/to/project/root/src` to `C:\\Users\\Username\\Documents\\bitcoin\\src`, since there have to understand that `root` means `bitcoin` and `src` refers to the literal `src` folder.

-----

My inspiration for using concrete examples was https://github.com/hebasto/bitcoin/pull/328/files#diff-4d2a64ce14cb8b971dbba9455421b04ae7ed0c489c66d983664be5632b0de4a3R19

-----

If it's not helpful, I don't mind reverting.

Reply:
Valid points, thanks @glozow & @maflcko, dropped that commit.